src/containers/timer/countdown/countdown-display/countdown-display.tsx

```diff
@@ -156,6 +172,15 @@ export default function CountdownDisplay() {
               className={timeInputClassName}
               readOnly={isActive}
             />
+            <Button
+              size="icon"
+              variant="primary-ghost"
+              className="max-md:hidden size-6 lg:size-12 rounded-full"
+              disabled={seconds <= 0}
```

Comment:
시간이 `03:00`일 때에도 초는 0이지만 초 감소는 할 수 있어야 하니까, `seconds` 값이 아닌 `leftTime` 값을 봐야 할 것 같네요!

```suggestion
              disabled={leftTime <= 0}
```